lib/bambora/client.rb

```diff
@@ -160,5 +170,13 @@ def www_form_client
         sub_merchant_id: sub_merchant_id,
       )
     end
+
+    def batch_payment_file_upload_client
+      Bambora::Rest::BatchPaymentFileUploadClient.new(
```

Comment:
I see other clients have been memoized, this one is not, method doesn't have arguments to make a different instance anyway.